google/resource_compute_target_ssl_proxy.go

```diff
@@ -41,31 +60,30 @@ func resourceComputeTargetSslProxy() *schema.Resource {
 					DiffSuppressFunc: compareSelfLinkOrResourceName,
 				},
 			},
-
-			"description": &schema.Schema{
+			"description": {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
 			},
-
-			"proxy_header": &schema.Schema{
+			"proxy_header": {
 				Type:     schema.TypeString,
 				Optional: true,
 				Default:  "NONE",
 			},
-
-			"project": &schema.Schema{
+			"creation_timestamp": {
 				Type:     schema.TypeString,
-				Optional: true,
 				Computed: true,
-				ForceNew: true,
 			},
-
-			"proxy_id": &schema.Schema{
+			"proxy_id": {
```

Comment:
Why do most things not have &schema.Schema, but `project` and `self_link` do?

Reply:
Different loop. I will fix it in a separate PR because it will cause changes to all resources.

Reply:
See #1377